docs/UserGuide.md

```diff
@@ -335,7 +335,7 @@ Notify the user if a certain stock is below threshold
 |**Add to Inventory**    | `add-i n/NAME q/QUANTITY [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]...​` <br> e.g., `add n/Chicken q/3 s/ShengSiong t/Poultry` |
 |**Clear from Inventory**  | `clear-i`                                                                                             |
 |**Delete from Inventory** | `delete-i INDEX`<br> e.g., `delete 3`                                                                 |
-|**Edit Inventory**   | `edit-i INDEX [n/NAME] [q/QUANTITY] [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]…​`<br> e.g.,`edit 1 n/Chicken q/50`                |
+|**Edit Inventory**   | `edit-i INDEX [n/NAME] [q/QUANTITY] [s/SUPPLIER] [max/MAX_QUANTITY] [t/TAG]…​`<br> e.g.,`edit 1r n/Chicken q/50`                |
```

Comment:
is there an extra r?

Reply:
@xnoobftw did the changes, thanks for spotting!